Gemfile

```diff
@@ -6,6 +6,9 @@ gem 'rails', '~> 4.2.4'
 # Use PostgreSQL as the database for Active Record
 gem 'pg'
 
+# InfluxDB for Github rate limit tracking
+gem 'influxdb-rails'
```

Comment:
Usually we lock down the version a little bit, but this is only `0.1.11`.  Should we lock down to `~> 0.1.11`?